locales/en.json

```diff
@@ -0,0 +1,20 @@
+{
+  "nav_communities": "Communities",
```

Comment:
I don't think we should prefix these strings. We started doing that in jerboa and it turned out to be a huge mistake, with 4 different versions of `_communities` all being the same word.

Reply:
i was concerned that the context of the word/phrase can affect the meaning and therefore the words being used, specially in other languages.

it sounds like your experience with jerboa indicates that is not a concern so i will remove the context